jwst/lib/dispaxis.py

```diff
@@ -0,0 +1,152 @@
+import logging
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+def get_dispersion_direction(exposure_type, grating="ANY", filter="ANY",
+                             pupil="ANY"):
+    """Get the dispersion direction.
+
+    Parameters
+    ----------
+    exposure_type : str
+        The exposure type.
+
+    grating : str
+        The name of the optic in the grating wheel.
+
+    filter : str
+        The name of the optic in the filter wheel.
+
+    pupil : str
+        The name of the optic in the pupil wheel.
+
+    Returns
+    -------
+    int : The dispersion direction
+
+-       0  the dispersion direction is not meaningful or not defined
+-       1  the dispersion direction is horizontal ("sky" coordinates)
+-       2  the dispersion direction is vertical ("sky" coordinates)
+    """
+
+    exposure_type = exposure_type.upper()
+    grating = grating.upper()
+    filter = filter.upper()
```

Comment:
It's a minor thing but could you change the name of this variable since it's a reserved python keyword.

Reply:
Yes; thanks for pointing this out.

Reply:
Is this a potential problem throughout all the cal pipeline code? I'm sure the variable name "filter" must be used other places, given what a common thing it is.